### PR TITLE
#MAR-1907 update domain.js

### PR DESCRIPTION
**Pull request by malmas18**
description is still missing

### CODE REVIEW

File: src/bundles/SellerRegistration/components/DomainSelector/domains.js [hunk 1]
```diff
@@ -32,7 +32,17 @@ export default [
     {
         key: 'change',
         label: 'Change, Training and Transformation',
-        description: 'Change, training and transformation is about embedding more efficient work culture and practices in government. Examples include digital transformation advice, agile coaching, change management and training in new ways of working.'
+        description: 'Change, training and transformation will soon be removed. New sellers should consider offering Change and Transformation and/or Training, Learning and Capability.'
```
Comment (tobiasbrown) [inline src/bundles/SellerRegistration/components/DomainSelector/domains.js]: `Training, Learning and Capability` or `Training, Learning and Development`?

File: src/bundles/SellerRegistration/components/DomainSelector/domains.js [hunk 2]
```diff
@@ -32,7 +32,17 @@ export default [
...
+    },
+    {
+        key: 'changetransformation',
+        label: 'Change and Transformation',
+        description: 'Change and transformation is about embedding more efficient work culture and practices in government. Examples include digital transformation advice, change management and transitioning to new ways of working.'
+    },
+    {
+        key: 'training',
+        label: 'Training, Learning and Development',
```
Comment (tobiasbrown) [inline src/bundles/SellerRegistration/components/DomainSelector/domains.js]: `Training, Learning and Capability` or `Training, Learning and Development`?

File: src/bundles/SellerRegistration/components/DomainSelector/domains.js (hunk 1 above)
Comment (malmas18) [reply src/bundles/SellerRegistration/components/DomainSelector/domains.js]: Supposed to be Training, Learning and Development

File: src/bundles/SellerRegistration/components/DomainSelector/domains.js (hunk 2 above)
Comment (malmas18) [reply src/bundles/SellerRegistration/components/DomainSelector/domains.js]: Supposed to be Training, Learning and Development

**Pull request merged**